File: Lições/python/EXERCICIO05.py
# 05) Escreva um algoritmo que receba o nome de uma pessoa e o ano de seu nascimento. Calcule 
# e exiba a idade e o nome da pessoa.

print('Quantos anos você tem?')
nome = str(input('Digite seu nome: '))
ano_nasc = int(input('Digite seu ano de nascimento: '))
print('Olá {} você está atualmente com {} anos de idade'.format(nome, 2022 - ano_nasc))

File: Lições/c/EXERCICIO05.c
// 05) Escreva um algoritmo que receba o nome de uma pessoa e o ano de seu nascimento. Calcule 
// e exiba a idade e o nome da pessoa.
#include <stdio.h>
#include <locale.h>
#include <conio.h>
char nome[100];
int ano;

int main(){
	setlocale(LC_ALL,"portuguese");

	printf("Qual o seu nome? ");
	gets(nome);
	printf("Quantos anos você tem? ");
	scanf("%i",&ano);
	
	printf("Olá %s você está atualmente com %i anos de idade",nome, 2022 - ano);

	return 0;
}
